packages/docs/celo-gold-holder-guide/quick-start.md

```diff
@@ -147,6 +148,8 @@ To allow you to keep your Beneficiary Ledger offline on a day-to-day basis, it
 
 This is a two step process. First, you create a "proof of possession" that shows that the holder of the beneficiary key also holds the vote signer key. Then, you will use that when the benficiary signs a transaction authorizing the vote signer key. This proves to the Celo network that a single entity holds both keys.
 
+A vote signer can either be another Ledger device or a cloud Hardware Security Module (HSM). Explore [this guide](cloud-hsm.md) to learn more about cloud HSM setup and celocli integration.
```

Comment:
Might include this in a hint or add some context to help people decide when to use cloud HSM over hardware wallet

Reply:
I added it as a hint. I'm hesitant to provide too much guidance of using one over another as I want the user to do their research and make the decision when they'd like to use it (similar liability concerns as self-custody).